static/js/containers/App_test.js

```diff
@@ -43,6 +54,65 @@ describe('App', () => {
     });
   });
 
+  describe('dashboard errors', () => {
+    let errorString = `Sorry, we were unable to load the data necessary
+      to process your request. Please reload the page.`;
+    errorString = errorString.replace(/\s\s+/g, ' ');
+
+    it('error from the backend triggers error message in dashboard', done => {
+      let response = _.cloneDeep(DASHBOARD_RESPONSE_ERROR);
```

Comment:
We shouldn't need to clone this here or below since we shouldn't be modifying this value in our actions or API functions